.github/workflows/test.yml

```diff
@@ -363,8 +365,9 @@ jobs:
       - name: rust cache
         uses: Swatinem/rust-cache@v2
         with:
-          prefix-key: "v1-${{ runner.os }}-rust-cache"
-          shared-key: "cli-e2e"
+          prefix-key: "v3"
```

Comment:
This these same prefix and shared key are identical for all steps, could we remove one of them and just use `rust-cache` for all?

Reply:
going to be closing PR because moving it all to the same cache had a negative effect